Logos/IPQualityScore_logo.svg

```diff
@@ -0,0 +1 @@
5.99 180.95 95.99 181.27 95.41 181.27 95.41 184.1 95.06 184.1"/><rect class="cls-3" x="97.58" y="180.95" width="0.35" height="3.15"/><polygon class="cls-3" points="99.91 180.95 100.42 183.69 100.93 180.95 101.25 180.95 100.66 184.1 100.14 184.1 99.55 180.95 99.91 180.95"/><polygon class="cls-3" points="104 182.34 104 182.66 103.22 182.66 103.22 183.78 104.17 183.78 104.17 184.1 102.87 184.1 102.87 180.95 104.17 180.95 104.17 181.27 103.22 181.27 103.22 182.34 104 182.34"/><polygon class="cls-3" points="105.84 184.1 105.84 180.95 106.2 180.95 106.2 183.78 107.1 183.78 107.1 184.1 105.84 184.1"/><polygon class="cls-3" points="108.99 182.97 108.33 180.95 108.69 180.95 109.18 182.5 109.68 180.95 110.01 180.95 109.34 182.97 109.34 184.1 108.99 184.1 108.99 182.97"/><path class="cls-3" d="M114.35,181c.5,0,.71.29.71.77v.38a.66.66,0,0,1-.74.76H114v1.24h-.35V181Zm0,1.59c.26,0,.39-.12.39-.42v-.42c0-.29-.1-.43-.36-.43H114v1.27Z"/><path class="cls-3" d="M117.38,181c.5,0,.71.25.71.72v.26c0,.34-.12.56-.39.64.28.09.39.32.39.66v.49a.92.92,0,0,0,.06.38h-.35a.82.82,0,0,1-.06-.38v-.5c0-.35-.17-.46-.46-.46H117v1.34h-.35V181Zm-.09,1.49c.28,0,.45-.09.45-.44v-.3c0-.28-.1-.43-.36-.43H117v1.17Z"/><polygon class="cls-3" points="120.93 182.34 120.93 182.66 120.16 182.66 120.16 183.78 121.1 183.78 121.1 184.1 119.81 184.1 119.81 180.95 121.1 180.95 121.1 181.27 120.16 181.27 120.16 182.34 120.93 182.34"/><polygon class="cls-3" points="123.04 180.95 123.56 183.69 124.07 180.95 124.39 180.95 123.8 184.1 123.28 184.1 122.69 180.95 123.04 180.95"/><polygon class="cls-3" points="127.13 182.34 127.13 182.66 126.36 182.66 126.36 183.78 127.31 183.78 127.31 184.1 126.01 184.1 126.01 180.95 127.31 180.95 127.31 181.27 126.36 181.27 126.36 182.34 127.13 182.34"/><polygon class="cls-3" points="129.29 184.1 128.98 184.1 128.98 180.95 129.42 180.95 130.15 183.23 130.15 180.95 130.46 180.95 130.46 184.1 130.1 184.1 129.29 181.55 129.29 184.1"/><polygon class="cls-3" points="132.63 184.1 132.63 181.27 132.05 181.27 132.05 180.95 133.56 180.95 133.56 181.27 132.98 181.27 132.98 184.1 132.63 184.1"/><polygon class="cls-3" points="138.27 182.42 138.27 182.74 137.55 182.74 137.55 184.1 137.19 184.1 137.19 180.95 138.45 180.95 138.45 181.27 137.55 181.27 137.55 182.42 138.27 182.42"/><path class="cls-3" d="M140.78,181c.49,0,.7.25.7.72v.26c0,.34-.12.56-.39.64.29.09.4.32.4.66v.49a.78.78,0,0,0,.06.38h-.36a.85.85,0,0,1-.05-.38v-.5c0-.35-.17-.46-.46-.46h-.25v1.34h-.35V181Zm-.09,1.49c.28,0,.45-.09.45-.44v-.3c0-.28-.11-.43-.37-.43h-.34v1.17Z"/><path class="cls-3" d="M143.56,183.47l-.12.63h-.33l.6-3.15h.51l.61,3.15h-.35l-.13-.63Zm0-.31h.7l-.35-1.81Z"/><path class="cls-3" d="M146.79,183.36c0,.28.11.45.36.45s.37-.17.37-.45V181h.33v2.39c0,.48-.22.8-.71.8s-.7-.32-.7-.8V181h.35Z"/><path class="cls-3" d="M150.28,181c.49,0,.72.3.72.78v1.59c0,.48-.23.78-.72.78h-.73V181Zm0,2.83c.25,0,.37-.16.37-.44v-1.62c0-.29-.12-.45-.37-.45h-.38v2.51Z"/><text class="cls-4" transform="translate(150.79 180.54)">TM</text></svg>
```

Comment:
@RamboV : Could you please fix the validation errors?
Validations are failing because the provided Logo does not adhere to the Logo guidelines. Please refer the guidelines mentioned [here](https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/ReadMe.md)

Thanks!!!

Reply:
@v-rucdu 
Validations are passed now. Thanks.